cmd/spire-agent/cli/run/run.go

```diff
@@ -96,6 +96,15 @@ func (*RunCLI) Run(args []string) int {
 		return 1
 	}
 
+	// Create uds dir and parents if not exists
+	dir := filepath.Dir(c.BindAddress.String())
+	c.Log.Infof("creating spire agent UDS path %s", dir)
```

Comment:
Logging should start with capital letter (i.e. `"Creating UDS directory"`)

Prefer fields for parameterized data (keeps the logs clean and easier to read) (i.e. `c.Log.WithField("dir", dir).Info("Creating UDS directory")`)